src/utils/get-edge-path/get-edge-path.ts

```diff
@@ -8,11 +9,16 @@ export function getEdgePath(): string {
       return '/Applications/Microsoft Edge.app/Contents/MacOS/Microsoft Edge';
 
     case 'Windows_NT':
-      return '%windir%/SystemApps/Microsoft.MicrosoftEdge_8wekyb3d8bbwe/MicrosoftEdge.exe';
+      return 'C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe';

```

Comment:
Hum, you have taken the easy path :)
Unless I am wrong, there are also x64 versions of msedge and you should always take the x64 first even when there is a win32 install. 

Let me check this point on my windows machine at work tomorrow.

Reply:
I did not use windows sine WindowsXP) And I do not have any Windows PC to check.

Reply:
The `appveyor` CI is there to run all tests on Windows :)

So even without having a windows machine in your hands you can still have a clue of how it runs on Windows by clicking  the `appveyor` link provided by the `appveyor` badge in the README.

Furthermore, if you add in the `appveyor.yml` the line to install msedge you will get the real install path :)